ChartboostMediationAdapterVungle.podspec

```diff
@@ -14,7 +14,7 @@ Pod::Spec.new do |spec|
 
   # Minimum supported versions
   spec.swift_version         = '5.0'
-  spec.ios.deployment_target = '10.0'
+  spec.ios.deployment_target = '11.0'
```

Comment:
We should update the min version in the [README](https://github.com/ChartBoost/chartboost-mediation-ios-adapter-vungle/blob/27d92c38f3580e394514eabbd4f9d23d738d2993/README.md?plain=1#L11) as well.

Reply:
Done.